INSTALL.md

```diff
@@ -221,7 +221,9 @@ Building static web resources
 -------------------------------
 
 To build everything necessary to access the server in a web browser (CSS,
-JavaScript), run the following command:
+JavaScript), follow this section [Translations](https://github.com/metabrainz/musicbrainz-server/blob/master/INSTALL.md#translations) first to generate files for translation.
+
+With the translation files, you are ready to run the following command to build static web resources:
```

Comment:
The translations are not mandatory. Actually [the default is to have no translations](https://github.com/metabrainz/musicbrainz-server/blob/v-2023-09-18/lib/DBDefs/Default.pm#L376-L379). Might it be you customized the value of `MB_LANGUAGES` in your file `lib/DBDefs.pm`? That would explain the error.

Reply:
Maybe the section about translations should just be moved above the section about building static resources.

Reply:
Sorry for the delayed response. I am actually inclined to **mentioning that readers need to finish the translation's `4. install translations` section first**, because the steps further are not required to build static web resources.

To answer your question, no, I didn't customize the value of `MB_LANGUAGES`. The lines below in my `lib/DBDef.pm` are commented out:

```
# Please activate the officially approved languages here. Not every .po
# file is active because we might have fully translated languages which
# are not yet properly supported, like right-to-left languages
# sub MB_LANGUAGES {qw()}
```

I'll paste the entire error message just so you know:

```
2023-09-21 20:35:03 node:fs:1713
2023-09-21 20:35:03   handleErrorFromBinding(ctx);
2023-09-21 20:35:03   ^
2023-09-21 20:35:03 
2023-09-21 20:35:03 Error: ENOENT: no such file or directory, stat '/app/app/musicbrainz-server/po/javascript.bg.po'
2023-09-21 20:35:03     at Object.statSync (node:fs:1713:3)
2023-09-21 20:35:03     at find (file:///app/app/musicbrainz-server/root/server/gettext/poFile.mjs:29:8)
2023-09-21 20:35:03     at Module.load (file:///app/app/musicbrainz-server/root/server/gettext/poFile.mjs:54:23)
2023-09-21 20:35:03     at createJsPo (file:///app/app/musicbrainz-server/webpack/client.config.mjs:171:26)
2023-09-21 20:35:03     at loadNewerPo (file:///app/app/musicbrainz-server/webpack/client.config.mjs:183:15)
2023-09-21 20:35:03     at file:///app/app/musicbrainz-server/webpack/client.config.mjs:210:27
2023-09-21 20:35:03     at Array.forEach (<anonymous>)
2023-09-21 20:35:03     at file:///app/app/musicbrainz-server/webpack/client.config.mjs:209:19
2023-09-21 20:35:03     at Array.forEach (<anonymous>)
2023-09-21 20:35:03     at file:///app/app/musicbrainz-server/webpack/client.config.mjs:202:14 {
2023-09-21 20:35:03   errno: -2,
2023-09-21 20:35:03   syscall: 'stat',
2023-09-21 20:35:03   code: 'ENOENT',
2023-09-21 20:35:03   path: '/app/app/musicbrainz-server/po/javascript.bg.po'
2023-09-21 20:35:03 }
```